recipes/text_extensions_for_pandas/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "text_extensions_for_pandas" %}
+{% set version = "0.1b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CODAIT/text-extensions-for-pandas/archive/{{ version }}.tar.gz
+  sha256: ab14b12dcea4b6928c2e1e5c83e3d23dd2b14baee1954504460ed93dd321405c
+
+build:
+  number: 0
+  skip: true  # [win,py<37]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.17
+    - pandas >=1.0.3,<1.2.0
+    - pyarrow >=1.0.0
+    - regex
+    - memoized-property  # [py36,py37]
```

Comment:
Do you mean `py<38`?

Reply:
I tried, but got the error:
```
Error: Invalid selector in meta.yaml line 29:
offending line:
    - memoized-property  # [<py38]
    - ```

Reply:
actually, this is still needed for all versions so this shouldn't be there